wiki-thread-lighting-app.md

```diff
@@ -0,0 +1,103 @@
+- Matter SDK: [`6b01cb9`](https://github.com/project-chip/connectedhomeip/commit/6b01cb977127eb8547ce66d5b627061dc2dd6c90) (API version: 6)
+- RCP firmware: [`00ac6cd`](https://github.com/openthread/ot-nrf528xx/tree/00ac6cd0137a4f09288b455bf8d7aa72d74062d1) (API version: 6)
+
+#### 1. Form a Thread Network on Machine A
+On Machine A, following this guide to [install and configure the OTBR snap](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#install-and-configure-the-otbr-snap) 
+and [form a Thread network](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#form-a-thread-network) 
+using the nRF52840 Dongle as the RCP. For details on setting up the RCP, 
+pelase refer to [Build and flash RCP firmware on nRF52480 dongle](https://github.com/canonical/openthread-border-router-snap/wiki/Setup-OpenThread-Border-Router-with-nRF52840-Dongle#build-and-flash-rcp-firmware-on-nrf52480-dongle).
+
+#### 2. Run a Matter Thread Lighting App on Machine B
+There are two options to run a Matter Thread lighting application. 
+
+**Option 1**: Native Build Lighting App
+
+You can choose this option to build and run the Matter Thread lighting app from the Matter SDK in the native shell. With this option, you can observe the lighting status (on/off) from the logs.
+
+To get started, follow these steps:
+
+- 1. Install the Matter SDK with a shallow clone for the Linux platform:
+```
+git clone https://github.com/project-chip/connectedhomeip.git --depth=1
+cd connectedhomeip
+git checkout 6b01cb977127eb8547ce66d5b627061dc2dd6c90
+scripts/checkout_submodules.py --shallow --platform linux
+```
+- 2. Build the lighting app:
+```
+cd examples/lighting-app/linux
+gn gen out/debug
+ninja -C out/debug
+```
+- 3. Run the lighting app with Thread feature enabled:
+```
+./out/debug/chip-lighting-app --thread
+```
+
+**Option 2**: GPIO Commander Snap
+
+Alternatively, you can use this option to set up the Matter Thread lighting application by installing the Matter Pi GPIO Commander snap. 
+This turns your Raspberry Pi into a Matter Thread lighting device, allowing you to control the connected LED, turning it on or off.
+
+Follow the instructions for installing the matter-pi-gpio-commander snap, 
+provided at [Setup OpenThread Border Router with nRF52840 Dongle](https://github.com/canonical/matter-pi-gpio-commander/wiki/Setup-and-control-a-lighting-device#installation).
+
+TBA: Then, running the GPIO commander snap with Thread feature enabled.
+
+#### 3. Run OTBR D-Bus Daemon on Machine B
+Check the [install and configure the OTBR snap](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#install-and-configure-the-otbr-snap) section
+to run OTBR dbus daemon, which is needed to support lighting app's Thread feature as pointed 
```

Comment:
The dbus daemon runs on the system. What is needed here is to run the OTBR agent. GUI is optional.

The lighting app communicates with OTBR agent via the dbus message bus.